### DISCUSSION TITLE
Git Branch icon isn't rendered properly

### DISCUSSION THREAD

**Discussion by ravishchawla** [starship/starship#1388] [Q&A] [answered]
The Git Branch Icon doesn't render properly. On the Mac, it shows a square with a question mark, and even installed on a linux server it has the same logo.

I am able to copy it, and this is what it renders as. The Git Logo is an unknown symbol:

`workspace/usergraph on  master [✘!+?] via 🐍 v2.7.16 on ☁️  us-west-2`

**Comment by vladimyr** [comment]
Which font do you use? Branch symbol is available only inside patched fonts aka nerd fonts.

**Comment by chipbuster** [comment, marked as answer]
The FAQ has a section on testing this: could you give the instructions there a go? If you don't see it render using those commands, it is a problem with your terminal/font/system.

https://starship.rs/faq/#why-don-t-i-see-a-glyph-symbol-in-my-prompt
